docker-compose-dev.yml

```diff
@@ -45,10 +79,17 @@ services:
 
   dev_smtp:
     container_name: "dev_smtp"
+    hostname: "postfix"
```

Comment:
Why "postfix"?

Reply:
This goes back to dataverse-k8s, where I used postfix for an MTA to get mail out. Is there a different name you'd prefer as a sensible default for not just dev environments?

Reply:
Thanks for the explanation - What about "smtp_server", or "mailserver"? We are not using a postfix container here.

Reply:
I just noticed this. Yeah, "postfix" is a bit misleading. Maybe just "smtp"? I have memories of smtp.whatever.com. Or we can worry about this in a future PR.

Reply:
Done.